### ISSUE TITLE
Gym and Pokestop icons not showing

### ISSUE THREAD

**Issue by Phyore** [AHAAAAAAA/PokemonGo-Map#811]
I have tried running this straight from the zip after installing php etc, but gyms and pokestops are not loading on the map

**Comment by mherceg** [comment]
Did you start it with -dp -dg flags?

**Comment by Mardiie** [comment]
Same problem here.
The rest seems to be working just fine.
I've placed -dp -dg flags in several locations in my line, though that should not matter. The system is finding gym's and stop's it's just not making them visible on the map.

**Comment by SkyBadPT** [comment]
They are not showing for me too and im using the flags:  `python runserver.py -a ptc -u USER -p PASS -l "LOCATION" -st 10 -H 00.00.00.00 -P 5000 -ar 5 -dl -dg `

**Comment by Mardiie** [comment]
I should note that when getting the last dev branch update the script already fails at line 9 so :P

**Comment by PokeDon** [comment]
I got the map up and running but nothing appears on the map. No pokemons, no pokestops, no gyms.
![capture](https://cloud.githubusercontent.com/assets/20577279/17020319/c9023fee-4f38-11e6-9117-386f5fc19296.JPG)

**Comment by Mardiie** [comment]
Pokémons work fine.

**Comment by DerScreever** [comment]
you need to pass -dp and -dg. work for me on latest develop

**Comment by PokeDon** [comment]
Tried -dp -dg also 

python runserver.py -a ptc -u [USERNAME] -p [PASSWORD] -l "LOCATION" -st 100 -dp -dg

Ive tried internet explorer and google chrome.

![capture2](https://cloud.githubusercontent.com/assets/20577279/17020487/c10191d6-4f39-11e6-9bb9-fa029004bb25.JPG)

**Comment by Mardiie** [comment]
PokeDon you are making a mistake with login probably since it works fine. Stop polluting the thread. 
DerScreever I find that hard to believe. 
My map is updating no stop. It's finding gyms and stops. Its Upserting? (nice word) them to the DB. But after that they are a no show on the map. 
Pokemon's work fine. It finds them, then downloads the icon and info and show up on the map.

**Comment by Mardiie** [comment]
![image](https://cloud.githubusercontent.com/assets/16288660/17020664/224ff5d2-4f43-11e6-84e0-db48089f5ff0.png)

**Comment by PokeDon** [comment]
Dude ur username and pw is showing lol

**Comment by Mardiie** [comment]
Lol fking CMD. tY DUDE. Luckily its app specific
Anyway im not sure what's going wrong with your map. 
Syntax seems fine, but maybe something wrong with the location data?
Are you giving in coordinates? 
Also don't use any " and [ but im sure your aware of that.

**Comment by SkyBadPT** [comment]
I still have problems is not showing if someone can help im trying with this command: `python runserver.py -a ptc -u USER -p PASS -l "LOCATION" -st 10 -H 00.00.00.00 -P 5000 -ar 5 -dl -dg`

**Comment by PokeDon** [comment]
lol change ur pw mate. yeah tried google api key. im using dev branch. Location data is good. Tried both forms of authentication, google and pokemon trainers club. I've even tried the ones in the wiki D.C and Washington. 

looks like i'm the only one with this issue. Maybe ill try at home. Maybe my work firewall is blocking it (i seriously doubt this though, i can get to any site.)

The only other things I have changed since I downloaded the dev branch is I renamed

credentials.example.json to credentials.json

and added the api key

{
	"gmaps_key" : "AIzaSyCtIs3sjA******DXfXxqqCNQ8sCw"
}

sigh.
